server/mscalendar/daily_summary_test.go

```diff
@@ -148,7 +148,7 @@ func TestProcessAllDailySummary(t *testing.T) {
 				gomock.InOrder(
 					mockPoster.EXPECT().DM("user1_mm_id", "You have no upcoming events.").Return("postID1", nil).Times(1),
 					mockPoster.EXPECT().DM("user2_mm_id", `Times are shown in Pacific Standard Time
-Wednesday February 12
+Wednesday February 12, 2020
```

Comment:
> is it a general improvement for both?

Yes, added in this PR as an improvement. It may have also just used it for debugging to know what year an event was. Sorting events was an issue at first and that made it easier to see the exact data.

https://github.com/mattermost/mattermost-plugin-mscalendar/blob/a88921f7bb191854adc2c1f54052eaa9d22919f6/server/mscalendar/views/calendar.go#L30